docker/files/provision-user

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+useradd ${KDK_USERNAME} -m -G wheel -s "${KDK_SHELL}"
+chown ${KDK_USERNAME} /home/${KDK_USERNAME}
+runuser -l ${KDK_USERNAME} -c "yadm clone --bootstrap ${KDK_DOTFILES_REPO}"
```

Comment:
I don't see "yadm" installed anywhere on the docker image.

Reply:
good catch.  typo in yum install.